Makefile

```diff
@@ -55,3 +65,31 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+.PHONY: bench-deps
+bench-deps:
+	@echo "Installing benchstat tool"
+	rm -rf ${BENCH_PATH}
+	mkdir ${BENCH_PATH}
+	go clean -testcache
+	cd ${BENCH_PATH} && git clone https://go.googlesource.com/perf && cd perf && go install ./cmd/benchstat
+	rm -rf ${BENCH_PATH}/perf
+
+.PHONY: bench
+${BENCH_FILE} bench:
+	@echo "Running benchmark tests from the current branch"
+	go test ${TEST_PATH} ${BENCH_OPTIONS} 2>&1 \
+		| tee ${BENCH_FILE}
+	benchstat ${BENCH_FILE}
+
+${REFERENCE_FILE}:
+	@echo "Running benchmark tests from master for using results in bench-diff target"
+	cd ${BENCH_PATH} && rm -rf go-tarantool && git clone https://github.com/tarantool/go-tarantool.git
```

Comment:
There are two things I'm still worrying about:
- target is not idempotent, we will clone repository every time when target will run. It looks inefficient. 
- target cloning repository that we already have. Probably we can use some git magic here. For example: git stash && git fetch && git checkout master && <MEASURE> && git checkout -`.

Reply:
1. I could leave the cloned repo and not removing it.
2. See https://github.com/tarantool/go-tarantool/pull/152#discussion_r852064168.